app/workers/invalidate_cache_worker.rb

```diff
@@ -0,0 +1,23 @@
+class InvalidateCacheWorker
+  include Sidekiq::Worker
+
+  def perform
+    creds = Aws::ECSCredentials.new(retries: 3)
+    client = Aws::CloudFront::Client.new(credentials: creds)
+
+    production_cdn = client.list_distributions.distribution_list.items.find { |d| d.comment = ENV.fetch('CDN_DISTRIBUTION_NAME', 'Production CDN') }
```

Comment:
```suggestion
    production_cdn = client.list_distributions.distribution_list.items.find { |d| d.comment = '#{ENV['ENVIRONMENT'].capitalize} CDN'}
```

Why introduce a new environment variable?